### PR TITLE
[windows] introduce dedicated PowerShell configuration step

### PR DESCRIPTION
**Pull request by ilia-shipitsin**
# Description

new "Configure-PowerShell" step is introduced which takes care of both system wide and
temporary user configuration

#### Related issue:

https://github.com/github/c2c-actions-akvelon/issues/38

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by erik-bershel — APPROVED** [review]

File: images/win/scripts/Installers/Configure-PowerShell.ps1 [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
+
+# Specifies the installation policy
+Set-PSRepository -InstallationPolicy Trusted -Name PSGallery
+#endregion
+
+#region User (current user, image generation only)
+if( -not(Test-Path $profile)) {
+    New-Item $profile -ItemType File -Force
+  }
```
Comment (vpolikarpov-akvelon) [inline images/win/scripts/Installers/Configure-PowerShell.ps1:17]: Please bring code into line with style guide (spaces in line 15 and indentation in line 17)

Comment (ilia-shipitsin) [reply images/win/scripts/Installers/Configure-PowerShell.ps1:17]: thanks, much better indeed. I'll submit a patch

File: images/win/scripts/Installers/Configure-PowerShell.ps1 [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#region System
+Write-Host "Setup PowerShellGet"
+Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
+
+# Specifies the installation policy
+Set-PSRepository -InstallationPolicy Trusted -Name PSGallery
+#endregion
+
+#region User (current user, image generation only)
+if( -not(Test-Path $profile)) {
```
Comment (vpolikarpov-akvelon) [inline images/win/scripts/Installers/Configure-PowerShell.ps1:15]:
```suggestion
if (-not (Test-Path $profile)) {
```

**Review by vpolikarpov-akvelon — APPROVED** [review]

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**